src/components/SHeaderItem.vue

```diff
@@ -0,0 +1,50 @@
+<script>
+  export default {
+    name: 'SHeaderItem'
+  }
+</script>
+
+<script setup>
+  const props = defineProps({
+    initials: {
+      type: String,
+      required: true
+    },
+    icon: {
+      type: Object,
+      required: true
+    }
+  })
+</script>
+
+<template>
```

Comment:
Not sure about my indentation here. The icon is also part of the clickable link and changes color on mouseover with the initials.